tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestReferenceStringSearch.java

```diff
@@ -37,11 +37,12 @@ public static final void main(String[] args) {
     }
 
     static final RuleBasedCollator TEST_COLLATOR =
-            (RuleBasedCollator) Collator.getInstance(ULocale.ENGLISH);
+            (RuleBasedCollator) Collator.getInstance(ULocale.ROOT); // freeze below
 
     static {
         TEST_COLLATOR.setStrength(Collator.PRIMARY);
         TEST_COLLATOR.setAlternateHandlingShifted(true); // ignore puncuation
+        TEST_COLLATOR.freeze();
```

Comment:
Could add to CollatorHelper: strength PRIMARY and setAlternateHandlingShifted(true); there is another above